src/ApparentHorizons/StrahlkorperGr.hpp

```diff
@@ -468,4 +468,14 @@ double irreducible_mass(double area);
  */
 double christodoulou_mass(double dimensionful_spin_magnitude,
                           double irreducible_mass);
+
+/*!
+ * Compute the pointwise radial distance between two Strahlkorpers with
+ * the same center, first interpolating `strahlkorper_b` to a Strahlkorper with
+ * the same angular resolution as `strahlkorper_a`.
+ */
+template <typename Frame>
+void radial_distance(gsl::not_null<Scalar<DataVector>*> radial_distance,
```

Comment:
Why a `Scalar<DataVector>` as opposed to just a `DataVector`? Is the result intended to be used with other tensors?

Reply:
In principle you could imagine using the result with other tensors, but I made this choice primarily for consistency with other functions in StrahlkorperGr.cpp and elsewhere. Others can correct me if I'm wrong, but my understanding is that in general it's preferable to pass Scalar<DataVector> instead of DataVector.

Reply:
That sounds good. Thank you for clarifying!